app/services/next_working_day_finder.rb

```diff
@@ -0,0 +1,20 @@
+require "business_time"
+
+class NextWorkingDayFinder
+
+  def self.run(date = Time.today)
+    self.new(date).run
+  end
+
+  def initialize(date = Time.today)
+    @date = date
```

Comment:
let's create an `attr_reader` for date. that's a style we've been using in the other service classes.